cli/cli.go

```diff
@@ -13,10 +13,11 @@ type Format struct {
 	WorkingDirectory      kong.ChangeDirFlag `default:"." short:"C" help:"Run as if treefmt was started in the specified working directory instead of the current working directory."`
 	NoCache               bool               `help:"Ignore the evaluation cache entirely. Useful for CI."`
 	ClearCache            bool               `short:"c" help:"Reset the evaluation cache. Use in case the cache is not precise enough."`
-	ConfigFile            string             `type:"existingfile" default:"./treefmt.toml" help:"The config file to use."`
+	ConfigFile            string             `type:"path" help:"Load the config file from the given path (defaults to finding treefmt.toml up)."`
 	FailOnChange          bool               `help:"Exit with error if any changes were made. Useful for CI."`
 	Formatters            []string           `short:"f" help:"Specify formatters to apply. Defaults to all formatters."`
-	TreeRoot              string             `type:"existingdir" default:"." help:"The root directory from which treefmt will start walking the filesystem."`
+	TreeRoot              string             `type:"path" help:"The root directory from which treefmt will start walking the filesystem (defaults to the directory containing the config file)."`
```

Comment:
You can use an `xor` in kong to ensure only one of these is set. 

https://github.com/alecthomas/kong?tab=readme-ov-file#supported-tags

Reply:
This would allow you to restore the `existingdir` type which adds a bit extra validation